spec/controllers/spree/admin/shipping_categories_controller_spec.rb

```diff
@@ -26,6 +26,15 @@ module Admin
           expect(response).to redirect_to spree.admin_shipping_categories_url
           expect(shipping_category.reload.name).to eq "Super Frozen"
         end
+
+        it "deletes an existing shipping category" do
+          shipping_category = create(:shipping_category)
+          expect {
+            spree_delete :destroy, id: shipping_category.id
+          }.to change(Spree::ShippingCategory.all, :count).by(-1)
```

Comment:
This is an outdated syntax. The `change` matcher should always use the block syntax now:
```suggestion
          }.to change { Spree::ShippingCategory.count }.by(-1)
```

Reply:
oh nice, I didn't know that syntax was deprecated :ok_hand:

Reply:
Ooo noce, I didn't realise the syntax could be different either 💡 I'll make the change @mkllnk 👍